include/nanorange/views/iota.hpp

```diff
@@ -303,6 +303,9 @@ struct iota_view : view_interface<iota_view<W, Bound>> {
           bound_(bound)
     {}
 
+    constexpr iota_view(iterator first, sentinel last)
+        : iota_view(*first, last.bound_) {}
```

Comment:
While others got their constructors trimmed, `iota` got a new one.

https://eel.is/c++draft/range.iota.view

Reply:
👍